daft/daft.pyi

```diff
@@ -34,4 +34,19 @@ class ImageMode(Enum):
         """
         ...
 
+class ImageFormat(Enum):
+    """
+    Supported image format for Daft's image I/O.
+    """
+
+    PNG: int
+    JPEG: int
```

Comment:
Starting small until we confirm the Rust `image` crate has good support for all listed formats.